.github/workflows/config.yml

```diff
@@ -16,6 +16,7 @@ jobs:
   run-tests:
     runs-on: ubuntu-latest
     name: Run Tests
+    continue-on-error: true
```

Comment:
whats this for

Reply:
thats supposed to make it optional I thought

Reply:
nvm I misunderstood the docs

Reply:
making it optional is just a repository setting

Reply:
ah, I dont have permission which is why I cant see the setting